packages/features/data-table/components/filters/ActiveFilters.tsx

```diff
@@ -21,7 +23,12 @@ export function ActiveFilters<TData>({ table }: ActiveFiltersProps<TData>) {
       {activeFilters.map((filter) => {
         const column = filterableColumns.find((col) => col.id === filter.f);
         if (!column) return null;
-        return <FilterPopover key={column.id} column={column} />;
+
```

Comment:
DateRangeFilter has popover functionality within itself, so I put it here like this.